docs/requirements/README.md

```diff
@@ -0,0 +1,4 @@
+## CLARIAH Requirements
```

Comment:
Is the path `docs/requirements` still accurate after the move to the `clariah-plus` repo? We now have top-level `use-cases/` so should we make `requirements/` top-level too?